youtube_dl/postprocessor/ffmpeg.py

```diff
@@ -278,6 +278,9 @@ def run(self, information):
 
         prefix, sep, ext = path.rpartition('.')  # not os.path.splitext, since the latter does not work on unicode in all setups
         new_path = prefix + sep + extension
+        
+        information['filepath'] = new_path
+        information['ext'] = extension
```

Comment:
But isn't `PostProcessingError` supposed to be raised if the converaion fails ? so it will never return on failure.